modules/misunderstood/src/backend/api.ts

```diff
@@ -65,10 +65,10 @@ export default async (bp: typeof sdk, db: Db) => {
     `/events/:status(${FLAGGED_MESSAGE_STATUSES.join('|')})`,
     asyncMiddleware(async (req: Request, res: Response) => {
       const { botId, status } = req.params
-      const { language, startDate, endDate } = extractQuery(req.query)
+      const { language, startDate, endDate, reason } = extractQuery(req.query)
```

Comment:
Does extractQuery validates the data ?

Reply:
It has the same behavior from analitycs' date filter, this was implemented in a previous pull request [4258](https://github.com/botpress/botpress/pull/4258). The unix time stamp will only be parsed if it's not undefined, later the date is checked at line 120 using Moment.js